tools/VscPreviewExtension/src/dfmService.ts

```diff
@@ -3,28 +3,26 @@
 import { AxiosError } from 'axios';
 
 import { DfmHttpClient } from './dfmHttpClient';
-import { DfmServiceResult } from './dfmServiceResult';
+import * as ConstVariable from "./ConstVariable";
 
 export class DfmService {
-    private static client = new DfmHttpClient();
-
-    static async previewAsync(content: String): Promise<DfmServiceResult> {
+    static async previewAsync(docfxServicePort, content: String, workspacePath, relativePath, writeTempPreviewFile = false, previewFilePath = null, pageRefreshJsFilePath = null, builtHtmlPath = null) {
         if (!content) {
             return null;
         }
 
-        return await DfmService.client.sendPostRequestAsync("preview", content);
+        return await DfmHttpClient.sendPostRequest(docfxServicePort, ConstVariable.previewCommand, content, workspacePath, relativePath, writeTempPreviewFile, previewFilePath, pageRefreshJsFilePath, builtHtmlPath);
     }
 
-    static async getTokenTreeAsync(content: String): Promise<DfmServiceResult> {
+    static async getTokenTreeAsync(docfxServicePort, content: String, workspacePath, relativePath) {
```

Comment:
remove other's type to kee consistence

---
In reply to: [111559443](https://github.com/dotnet/docfx/pull/1547#discussion_r111559443) [](ancestors = 111559443)